cloudnet-wrapper-jvm/src/main/resources/lang/chinese.properties

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2019-2021 CloudNetService team & contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# CloudNet language file
+# Language: english
+#
+# Node ModuleProviderHandler messages
```

Comment:
Why is this one not translated but the other ones?

Reply:
already solved.